config/form_profile_mappings/10-10EZR.yml

```diff
@@ -5,7 +5,7 @@ veteranDateOfBirth: [identity_information, date_of_birth]
 homePhone: [contact_information, us_phone]
 veteranAddress: [contact_information, address]
 email: [contact_information, email]
-# providers: [ezr_data, providers]
+providers: [ezr_data, providers]
```

Comment:
For our V2 flow, we will be utilizing `providers` and `dependents` to prefill the EZR.